.run/all-tests.run.xml

```diff
@@ -0,0 +1,12 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Is it possible to have this be placed into the `.idea` directory since it's contextually a feature in IntelliJ?

Reply:
I assume this file got ignored by `.gitignore`, might need to `!` to unignore that section

Reply:
For this, open another PR instead after this get merged

Reply:
Missed that, thanks for catch